Backend/src/swagger.json

```diff
@@ -1286,6 +1286,35 @@
         }
       }
     },
+    "/users/{id}": {
+      "get": {
+        "summary": "Get user by id",
+        "tags": [
+          "User"
+        ],
+        "parameters": [
+          {
+            "in": "path",
+            "name": "id",
+            "required": true,
+            "schema": {
+              "type": "string"
+            }
+          }
+        ],
+        "responses": {
+          "200": {
```

Comment:
As a good practice, you should write the user object with its fields and corresponding datatypes.